nginx/requirements.txt

```diff
@@ -0,0 +1 @@
+# integration pip requirements
```

Comment:
Yes, so here's the reason I think we should avoid adding requirements in the integrations that are shared with the core agent:

Given the way we currently build the integration packages we would end up with 2 copies of the same pip dependency (one in the main agent `site-packages` dir and one in the integration's dir), and there's absolutely no guarantee that the 2 copies will be at the same version of the dep.
Even though I _think_ python would always load the module in the agent `site-packages` dir first and ignore the one in the integration's dir I'm not very comfortable with having this additional copy of the module with a different version, it could potentially lead to weird behaviors at runtime.

By the way I think a similar issue could happen when the integration's pip sub-dependencies are shared with some of the agent's pip deps/sub-deps. At runtime we could end up with a version of a sub-dep module loaded by the python runtime which conflicts with the version that the integration's first-level dependency is expecting.

Hope this makes sense, and let me know if you think my concerns are valid or not :)